lab-yunjoo/test/storage-test.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+const expect = require('chai').expect;
+const Storage = require('../lib/storage');
+const testStorage = new Storage(`${__dirname}/data`);
```

Comment:
You are building a new data directory inside of your test folder, which is fine, but because there isn't a note directory in there, all three of these tests fail. That whole folder is gitignored, so it will never appear in a freshly cloned version of this project--you can verify that for yourself by cloning your project down into a different place and trying to run your tests. [The first comment on this thread is a good explanation of how to avoid problems like that in the future](http://stackoverflow.com/questions/18930103/git-ignore-everything-in-a-directory-except-subfolders).